linode/manager
# refactor: [M3-6286] - SRC > Features > TopMenu

## Description 📝
MUI v5 migration for `TopMenu`. 

## Major Changes 🔄
- Ran `tss-react` codemod on `TopMenu`
- Replaced `default export` with `named export`
- Deleted `index.tsx` files
- Convert Class Components to Functional Components

## How to test 🧪
1. Inspect the `TopMenu` and encompassing components for visual regressions.
2. Complete the inspection in Step 1 under both themes, light mode and dark mode.
3. Complete the inspection in Step 1 using browser devtools for mobile devices.
4. Ensure `TopMenu` functionality works as expected.
